templates/cli/lib/parser.js.twig

```diff
@@ -163,6 +163,7 @@ const commandDescriptions = {
     "locale": `The locale command allows you to customize your app based on your users' location.`,
     "projects": `The projects command allows you to view, create and manage your {{ spec.title|caseUcfirst }} projects.`,
     "storage": `The storage command allows you to manage your project files.`,
+    "messaging": `The messaging command allows you to manage and send messages to your users`,
```

Comment:
we have to remove 1.5 related items from here. The fixed CLI needs a release before 1.5